src/components/posts/PostsPage.js

```diff
@@ -14,11 +14,12 @@ import { Alert } from '@mui/material';
 import { useLoggedInUser } from '../../context/LoggedInUser';
 
 const PostsPage = ({ getUserId }) => {
-  const postPref = useLoggedInUser().user.postPrivacyPref === 'PRIVATE' ? true : false
+  const postPref =
+    useLoggedInUser().user.postPrivacyPref === 'PRIVATE' ? true : false;
```

Comment:
Rather than move this to a new line, I think it should be simplified to just

```js
const postPref = useLoggedInUser().user.postPrivacyPref === 'PRIVATE'
```

Reply:
my bad on this one 🤣

Reply:
made the change in a new branch going to merge now so the feature can be used